kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -262,18 +271,27 @@ async def set(self, *, updates: Collection[EntryUpdate], **rpc_kwargs) -> None:
             raise VSSClientError.from_grpc_error(exc) from exc
         self._process_set_response(resp)
 
-    async def authorize(self, *, token: str) -> str:
-        raise NotImplementedError('"authorize" is not yet implemented')
+    async def authorize(self, *, token: Dict) -> None:
+        self.authorization_header = "Bearer " + token
+
+    async def generate_metadata_header(self, metadata: list) -> list:
+        if metadata != None:
+            metadata = dict(metadata)
+        else:
+            metadata = dict()
+        metadata = metadata | {"authorization": self.authorization_header}
+        return list(metadata.items())
```

Comment:
This doesn't work for me
```
TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
```

```suggestion
        if metadata:
            metadata = dict(metadata)
        else:
            metadata = dict()
        metadata["authorization"] = self.authorization_header
        return list(metadata.items())
```

Reply:
Interesting. It's a merge operator. Which python version are you using?

Reply:
Ok, haven't seen it before.

I'm running python 3.8.10 (Ubuntu LTS 20.04)

Reply:
Either way, it's not even needed here as we're not merging dicts.